localdev/scripts/armada-start.sh

```diff
@@ -0,0 +1 @@
+go run ./cmd/armada/main.go --config ./localdev/config/armada/base.yaml --config ./localdev/config/armada/pulsar.yaml
```

Comment:
Again, do we want to use env vars or similar to support custom config? For example, my armada_jupyter stuff needs an option to be changed in the executor

Reply:
I agree.  I think its fine to have dev input their own custom configs if they need to make changes or testing.